processor/tailsamplingprocessor/README.md

```diff
@@ -50,7 +51,40 @@ processors:
             name: test-policy-4,
             type: rate_limiting,
             rate_limiting: {spans_per_second: 35}
-         }
+         },
+         {
+             name: composite-policy-1,
+             type: composite,
+             composite:
+               {
+                 max_total_spans_per_second: 1000,
+                 policy_order: [test-composite-policy-1, test-composite-policy-2],
```

Comment:
We can but then making it explicit with policy name makes it easy to read when we have many policies.

Reply:
That's arguable: a best practice would be to always keep the two lists in the same order, and seeing a policy in the map being declared after another policy would imply that it comes later in the chain: I would have to confirm this expectation by checking the policy_order. How about we leave this field out for this first implementation, and add it in case we have actual user demand for it ?

Reply:
sure